pages/linux/caja.md

```diff
@@ -0,0 +1,24 @@
+# caja
+
+> Manages files and directories in MATE desktop environment.
+> More information: <https://manned.org/caja.1>.
+
+- Open a current user home directory:
```

Comment:
```suggestion
- Open and display the home directory for the current user:
```

Reply:
I've used just: `Open the current user home directory`.